data/multipass.gui.desktop

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=Multipass
 Exec=multipass.gui
-Icon=${SNAP}/meta/gui/multipass-gui.svg
+Icon=multipass.gui.svg
```

Comment:
```
 $ desktop-file-validate data/multipass.gui.desktop
data/multipass.gui.desktop: error: (will be fatal in the future): value "multipass.gui.svg" for key "Icon" in group "Desktop Entry" is an icon name with an extension, but there should be no extension as described in the Icon Theme Specification if the value is not an absolute path
```

It will be a path in the snap, but not outside, and since this PR is to make it snap independent... Perhaps the extension should be added in the `sed` line, together with the path.